.changes/unreleased/NOTES-20250703-163653.yaml

```
@@ -0,0 +1,5 @@
+kind: NOTES
+body: These definitions are considered experimental and may change up until general availability.
```

Just a drive-by comment 🚗 , since we're also queuing up other protocol changes like https://github.com/hashicorp/terraform-plugin-go/pull/534, we should probably be more specific on what definitions are considered experimental.

```suggestion
body: The list resource protocol definitions are considered experimental and may change up until general availability.
```